composer.json

```diff
@@ -10,10 +10,12 @@
         "guzzlehttp/guzzle": "^7.2",
         "laravel/framework": "^10.10",
         "laravel/sanctum": "^3.2",
-        "laravel/tinker": "^2.8"
+        "laravel/tinker": "^2.8",
+        "livewire/livewire": "^3.0"
```

Comment:
Is this required for authorization?

Reply:
Laravel tinker is a command line to interact with Laravel ORM installed by Laravel Breeze. I think this utility could be helpful sometimes. 
Livewire is neccesary to react components, but we can not added yet on this PR.